examples/models/llama2/quantize.py

```diff
@@ -571,10 +571,9 @@ def dequantize_per_channel_group(
 
     w_int8_grouped = w_int8.reshape(-1, group_size)
     scales = scales.reshape(-1, 1)
-    zero_points = zero_points.reshape(-1, 1)
-    w_dq = (
-        w_int8_grouped.sub(zero_points).mul(scales).reshape_as(w_int8).to(output_dtype)
-    )
+    zp = zero_points.reshape(-1, 1) if zero_points is not None else 0
+    w_dq = w_int8_grouped.sub(zp).mul(scales).reshape_as(w_int8).to(output_dtype)
```

Comment:
`zp` is not used right now btw, can you add a TODO here so we don't forget to add it